launchdarkly/assets/dashboards/launchdarkly.json

```
@@ -3,79 +3,149 @@
-        "url_dark_theme": "https://static.launchdarkly.com/integrations/datadog/dashboard/header.png",
-        "sizing": "cover",
-        "has_background": false,
-        "has_border": false,
-        "vertical_align": "center",
-        "horizontal_align": "center"
-      },
-      "layout": { "x": 0, "y": 0, "width": 12, "height": 2 }
-    },
-    {
-      "id": 3385727926662895,
-      "definition": {
-        "title": "LaunchDarkly",
-        "title_size": "16",
-        "title_align": "left",
-        "type": "custom",
-        "app_id": "7144d0c5-42f2-4cc5-b562-5f77debc0c52",
-        "custom_widget_key": "launchdarkly",
-        "options": {
-          "dashboardURL": "https://app.launchdarkly.com/default/production/features",
-          "envTemplateVar": "ld-environment"
-        }
-      },
-      "layout": { "x": 0, "y": 2, "width": 5, "height": 7 }
-    },
-    {
-      "id": 3140411049621601,
-      "definition": {
-        "title": "LaunchDarkly feature flag events",
-        "title_size": "16",
-        "title_align": "left",
-        "type": "event_stream",
-        "query": "sources:launchdarkly tags:environment_key:$ld-environment",
-        "tags_execution": "and",
-        "event_size": "l"
+        "title": "Overview",
+        "background_color": "gray",
+        "show_title": true,
+        "type": "group",
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 6051850329724523,
+            "definition": {
+              "type": "image",
+              "url": "https://static.datadoghq.com/static/images/logos/launchdarkly_large.svg",
+              "url_dark_theme": "https://static.datadoghq.com/static/images/logos/launchdarkly_reversed_large.svg",
+              "sizing": "cover",
+              "margin": "md",
+              "has_background": true,
+              "has_border": true,
+              "vertical_align": "center",
+              "horizontal_align": "center"
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 6,
+              "height": 3
+            }
+          },
+          {
+            "id": 3979461125263392,
+            "definition": {
+              "type": "note",
+              "content": "### Track Change Events \nThe LaunchDarkly events integration for Datadog brings flag event markers to your monitoring dashboards, so you can see the effects of your LaunchDarkly feature deployments on your customer's services or systems.\n\n### Toggle Feature flags\nLaunchDarkly's feature flag tracking integration enriches your RUM data with your feature flags to provide visibility into performance monitoring and behavioral changes. \n\n### Relay proxy metrics \nIf you are using the LaunchDarkly Relay Proxy, you can configure it to export metrics, such as active and cumulative connections, to Datadog.\n\n[Learn more about the LaunchDarkly Integration](https://launchdarkly.com/docs/integrations/datadog)",
```

Assuming if applies to multiple customers, but ignore if this is not the case.

```suggestion
              "content": "### Track Change Events \nThe LaunchDarkly events integration for Datadog brings flag event markers to your monitoring dashboards, so you can see the effects of your LaunchDarkly feature deployments on your customers' services or systems.\n\n### Toggle Feature flags\nLaunchDarkly's feature flag tracking integration enriches your RUM data with your feature flags to provide visibility into performance monitoring and behavioral changes. \n\n### Relay proxy metrics \nIf you are using the LaunchDarkly Relay Proxy, you can configure it to export metrics, such as active and cumulative connections, to Datadog.\n\n[Learn more about the LaunchDarkly Integration](https://launchdarkly.com/docs/integrations/datadog)",
```